src/hooks/useTransactionHistoryWHScan.ts

```diff
@@ -301,6 +301,16 @@ const useTransactionHistoryWHScan = (
     [parseSingleTx],
   );
 
+  // Parser for Multi Token NTT transactions (appId === MTT)
```

Comment:
Curious, do these ever require customization and why not adjust when needed? Seems we are just duplicating references/callbacks here and above and below.